### PR TITLE
add df oneapi examples

### PR DESCRIPTION
**Pull request by makart19**
Added df oneapi examples for classification and regression

### CODE REVIEW

File: examples/sycl/decision_forest_classification_batch.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+                                                             varImportance='MDI',
+                                                             bootstrap=True,
+                                                             resultsToCompute='computeOutOfBagError',
+                                                             method=method
+                                                             )
+    # Training result provides (depending on parameters) model, outOfBagError, outOfBagErrorPerObservation and/or variableImportance
+    train_result = train_algo.compute(train_data, train_labels)
+
+    # now predict using the model from the training above
+    if daal_version < (2020,1):
```
Comment (PetrovKP) [inline examples/sycl/decision_forest_classification_batch.py:67]: Need delete it. We save version check for previous release

File: examples/sycl/decision_forest_classification_batch.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+    return (train_result, predict_result, predict_labels)
+
+
+if __name__ == "__main__":
+    (train_result, predict_result, plabels) = main()
+    print("\nVariable importance results:\n", train_result.variableImportance)
+    print("\nOOB error:\n", train_result.outOfBagError)
+    print("\nDecision forest prediction results (first 10 rows):\n", predict_result.prediction[0:10])
+    if daal_version >= (2020,1):
```
Comment (PetrovKP) [inline examples/sycl/decision_forest_classification_batch.py:140]: delete

File: examples/sycl/decision_forest_classification_batch.py [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+    # It is possible to specify to make the computations on GPU
+    if gpu_available:
+        with gpu_context():
+            sycl_train_data = sycl_buffer(train_data)
+            sycl_train_labels = sycl_buffer(train_labels)
+            sycl_predict_data = sycl_buffer(predict_data)
+            train_result, predict_result = compute(sycl_train_data, sycl_train_labels, sycl_predict_data, 'hist')
+            assert predict_result.prediction.shape == (predict_labels.shape[0], 1)
+            assert (np.mean(np.not_equal(predict_result.prediction - predict_labels, 0)) < 0.03).any()
```
Comment (PetrovKP) [inline examples/sycl/decision_forest_classification_batch.py:130]:
```suggestion
            assert (np.mean(predict_result.prediction == predict_labels)) < 0.03).any()
```

File: examples/run_examples.py [hunk 4]
```diff
@@ -108,6 +108,8 @@ def check_library(rule):
 req_version['sycl/linear_regression_batch.py'] = (2021,'P',110) # hangs in beta08, need to be fixed
 req_version['sycl/kmeans_batch.py'] = (2021,'P',200) # not equal results for host and gpu runs
 req_version['sycl/svm_batch.py'] = (2021,'B',107)
+req_version['sycl/decision_forest_classification_batch.py'] = (2021,'B',110)
```
Comment (PetrovKP) [inline examples/run_examples.py:111]: delete it.

File: examples/sycl/decision_forest_classification_batch.py (hunk 1 above)
Comment (makart19) [reply examples/sycl/decision_forest_classification_batch.py]: thanks, removed

File: examples/sycl/decision_forest_classification_batch.py (hunk 2 above)
Comment (makart19) [reply examples/sycl/decision_forest_classification_batch.py]: thanks, removed

File: examples/run_examples.py (hunk 4 above)
Comment (makart19) [reply examples/run_examples.py]: thanks, removed

File: examples/sycl/decision_forest_classification_batch.py (hunk 3 above)
Comment (makart19) [reply examples/sycl/decision_forest_classification_batch.py]: thank you, replaced with np.mean(predict_result.prediction != predict_labels)

**Review by PetrovKP — APPROVED** [review]

**Pull request merged**